tools/rna_tools/vienna_rna/rna2dfold.xml

```diff
@@ -16,63 +18,96 @@
         #if $varExists('$pfselect.pfscaling')
             --partfunc --pfScale=$pfselect.pfscaling
         #end if
-        #if $varExists('$advancedOptions.noconversion')
-            #if $advancedOptions.noconversion
-                --noconv
-            #end if
-            #if $advancedOptions.nogu
-                --noGU
-            #end if
-            #if $advancedOptions.noclosinggu
-                --noClosingGU
-            #end if
-            #if $advancedOptions.notetra
-                --noTetra
-            #end if
-            --maxDist1=$maxK
-            --maxDist2=$maxL
+        $noBT
+
+        $advancedOptions.noconversion
+		$advancedOptions.noclosinggu
+		$advancedOptions.circular
+    
+        #if $varExists('$advancedOptions.maxK')    
+            --maxDist1=$advancedOptions.maxK
+            --maxDist2=$advancedOptions.maxL
         #end if
+
         < $custom_input > $out_file
```

Comment:
Please also remember to put all type="data" and all type="text" into quotes. Like here `'$custom_input'`